css/fonts.css

```diff
@@ -0,0 +1,19 @@
+@font-face {
+font-family: 'Gotham Pro Regular';
+src: url('../fonts/GothamPro.eot');
+src: url('../fonts/GothamPro.eot?#iefix') format('embedded-opentype'),
+url('../fonts/GothamPro.woff') format('woff'),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)